proto/viam/common/v1/common.proto

```diff
@@ -131,7 +131,7 @@ message Transform {
   string reference_frame = 1;
   // the pose of the above reference frame with respect to a different observer reference frame
   PoseInFrame pose_in_observer_frame = 2;
-  optional Geometry geometry = 3;
+  optional Geometry physical_objects = 3;
```

Comment:
If we're going this route, it should be `physical_object` as `Geometry` has no way of being repeated

Reply:
done